mission-vision-values.md

```diff
@@ -139,14 +139,13 @@ other community members.
 It’s no secret that a good number of maintainers of the project are employed by
 companies with commercial interests in OpenTelemetry, especially vendors in the
 observability space. That said, we expect community members to act in the best
-interests of the project. Each member’s priorities can (and should!) align with
-those of their employers so that the relationship is beneficial to all parties,
-but when acting as a maintainer or contributor to the project, community members
-are expected to wear the project’s hat.
+interests of the project. Each member’s priorities when acting as a maintainer
+or contributor for the project should represent the interests of the project
+and its community first before their other obligations (e.g. employer).
```

Comment:
```suggestion
and its community first before their own interests and other obligations (e.g. employer).
```
I suggest we also explicitly add that they should put the interest of the project before their own (even employer-independent) interest.

Reply:
Agreed. Adding personal interest here is a good addition.